sorc/ncep_post.fd/RQSTFLD.F

```diff
@@ -2364,9 +2364,18 @@ module RQSTFLD_mod
      &                      /1,'F16 SSMIS NON-NADIR ',118,109,     &
      &                       'SSMIS TB top_of_atmos '/ !table 133
 
-!    SSMIS_F17 (L(1)-L(7) -> ID8: 201-207 -> 183H, 19H, 19V, 37H, 37V, 85H, 85V)
+!    SSMIS_F17 (SSMIS_F17 (CH15:37H, CH16:37V, CH17:91H, CH18:91V))
```

Comment:
@LinZhu-NOAA Can you also add a change log in this routine? Thanks!

Reply:
@WenMeng-NOAA The change log is added in this subroutine. Please check. Thanks!